tests/io/feeding_services/http_tests.py

```diff
@@ -70,12 +69,19 @@ def test_get_expired_token(self):
         self.assertEquals('', TestFeedingService()._get_auth_token(provider))
 
     def test_fetch_token(self):
+        # TODO: need some rewriting
```

Comment:
those credentials we were using don't work anymore, so you can probably get rid of it